portal-kernel/test/unit/com/liferay/portal/kernel/util/SystemPropertiesTest.java

```diff
@@ -58,6 +61,31 @@ public void testGetProperties() {
 		Assert.assertEquals(_TEST_VALUE, properties2.get(_TEST_KEY));
 	}
 
+	@Test
+	public void testGetPropertiesWithPrefix() {
+		String prefix =
+			SystemPropertiesTest.class.getName() + StringPool.PERIOD;
+
+		Map<String, String> propertiesWithPrefix =
+			SystemProperties.getProperties(prefix, false);
+
+		Assert.assertTrue(propertiesWithPrefix.isEmpty());
+
+		SystemProperties.set(_TEST_KEY, _TEST_VALUE);
```

Comment:
@W2jx It is not that obvious the _TEST_KEY's prefix is `SystemPropertiesTest.class.getName() + StringPool.PERIOD`